src/api.ts

```diff
@@ -61,6 +62,17 @@ export async function addDependency(
 
   const names = Array.isArray(name) ? name : [name];
 
+  if (resolvedOptions.packageManager.name === "deno") {
+    for (let i = 0; i < names.length; i++) {
+      if (!/^.*?:.+$/.test(names[i])) {
```

Comment:
Can we please restrict it? (i guess to `npm:` / `jsr:`)

Reply:
you're right, maybe also `file:` for local development? I can't see any valid value specification here https://docs.npmjs.com/cli/v10/using-npm/package-spec#aliases